index.js

```diff
@@ -1,155 +1,103 @@
-// Variables to call Elements by ID.
-const showFormBtn = document.getElementById("showFormBtn"),
-    form = document.getElementById("bookForm"),
-    libraryDivContainer = document.getElementById("renderLibrary"),
-    removeBookBtnQuery = document.getElementsByClassName("deleteBookBtn"),
-    openModalTrigger = document.querySelector(".trigger"),
-    closeModalTrigger = document.querySelector(".close"),
-    modal = document.querySelector(".modal")
-
-// Initialize the Library Array and new book variable.
-let myLibrary = []
-let newBook
-let tempLibrary
-
-// Book Constructor:
-function Book(title, author, pages, read = false) {
-    this.title = title,
-    this.author = author,
-    this.pages = pages,
-    this.read = read
-}
-
-// Modal EventListeners for Open, Close, and click outside Modal.
-openModalTrigger.addEventListener("click", () => modal.classList.add("open"))
-closeModalTrigger.addEventListener("click", () => modal.classList.remove("open"))
-window.addEventListener("click", (event) => {
-    if(event.target === modal) {
-        modal.classList.remove("open")
+const myLibrary = [];
```

Comment:
## Modularise the codebase
- [ ] kindly try and keep the `application logic` separated from `DOM manipulation` tasks ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/dom_logic_separation_requirement.png) from Pathwright).